pxtblocks/blocklyloader.ts

```diff
@@ -716,7 +716,8 @@ namespace pxt.blocks {
         block.setPreviousStatement(!(hasHandlers && !fn.attributes.handlerStatement) && fn.retType == "void");
         block.setNextStatement(!(hasHandlers && !fn.attributes.handlerStatement) && fn.retType == "void");
 
-        block.setTooltip(/^__/.test(fn.namespace) ? "" : fn.attributes.jsDoc);
+        block.setTooltip((/^__/.test(fn.namespace) ? "" : fn.attributes.jsDoc) + (pxt.blocks.showBlockIdInTooltip ? " (id: '" + fn.attributes.blockId + "')" : ""));
```

Comment:
This is a question about validation since I know that this tooltip and the flag are largely for help with investigation/debugging there. For our blocks exist validation currently, are we only looking at block id? Just wanted to make sure that there weren't other identifiers that we were using to look for blocks because I remember in some docs work that I did early on, that qName was used, but I might be misremembering it's significance.

Reply:
I believe we check against the id and nothing else, though we could potentially change that in the future if needed.